collect_app/src/androidTest/java/org/odk/collect/android/utilities/CollectServerClientTest.java

```diff
@@ -0,0 +1,112 @@
+package org.odk.collect.android.utilities;
+
+import org.junit.Before;
+import org.junit.Test;
+import org.odk.collect.android.http.CollectServerClient;
+import org.odk.collect.android.http.HttpClientConnection;
+import org.odk.collect.android.test.MockedServerTest;
+import okhttp3.mockwebserver.MockResponse;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNull;
+import static org.odk.collect.android.test.TestUtils.assertMatches;
+
+public class CollectServerClientTest extends MockedServerTest {
+
+    private CollectServerClient collectServerClient;
+
+    @Before
+    public void setUp() throws Exception {
+        // server hangs without a response queued:
+        server.enqueue(new MockResponse());
+        collectServerClient = new CollectServerClient(new HttpClientConnection());
+    }
+
+    //    @Test
+    //    public void httpRequests_shouldHaveUseragentHeader() throws Exception {
+    //        // when
+    //        doRequest("/some-path");
+    //
+    //        // then
+    //        RecordedRequest r = nextRequest();
+    //        assertEquals("GET /some-path HTTP/1.1", r.getRequestLine());
+    //        assertTrue(r.getHeader("User-Agent").matches("Dalvik/.* org.odk.collect.android/.*"));
+    //    }
```

Comment:
Commented-out code

Reply:
I've reinstated this test so that it is no longer commented out.